docs/tutorials/platform_beginner/3_customize_the_front_page.md

```diff
@@ -233,7 +233,7 @@ Clear the cache and regenerate the assets by running the following commands:
 ``` bash
 php bin/console cache:clear
 php bin/console assets:install
-yarn encore prod
+yarn encore <dev|prod>
```

Comment:
I would explain below these two options are different (or if/why one should use dev for the purpose of this tutorial).